### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-tf-module.git`](https://github.com/cisagov/skeleton-tf-module.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/cert-read-role-tf-module.git cert-read-role-tf-module
    cd cert-read-role-tf-module
    git remote add skeleton https://github.com/cisagov/skeleton-tf-module.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the branch,
   `commit`, and `push` your changes:

    ```console
    git add README.md outputs.tf variables.tf 
    git commit --message="Resolve Lineage conflicts."
    git push --force --set-upstream origin lineage/skeleton
    ```

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,20 +54,33 @@ This meta-role requires a permission policy similar to the following:
...
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-------:|:--------:|
-| account_ids | AWS account IDs that are to be allowed to assume the role. | list(string) | [] | no |
-| cert_bucket_name | The name of the AWS S3 bucket where certificates are stored. | string | | yes |
-| cert_path | The path to the certificates in the AWS S3 bucket.  For example, the certificate files for site.example.com are expected to live at <cert_bucket_path>/site.example.com/*. | string | "live" | no |
-| hostname | The FQDN corresponding to the certificate to be read (e.g. site.example.com). | string | | yes |
+| account_ids | AWS account IDs that are to be allowed to assume the role | `list(string)` | `[]` | no |
+| cert_bucket_name | The name of the AWS S3 bucket where certificates are stored | `string` | n/a | yes |
+| cert_path | The path to the certificates in the AWS S3 bucket.  For example, the certificate files for site.example.com are expected to live at <cert_path>/site.example.com/* | `string` | "live" | no |
```
Comment (mcdonnnj) [inline README.md]: I think `"live"` should be `` `live` ``? That's what the command I've been using outputs (I dug that up on Chat). Command I've been using for reference:
``terraform-docs markdown --no-escape --no-header . | sed -E 's/`"|"`/`/g' ``

**Review by mcdonnnj — APPROVED** [review]
🤖 🌟

File: README.md (hunk 1 above)
Comment (dav3r) [reply README.md]: Ah thanks- I haven't been using `terraform-docs` lately, but I will go back to it now.  

See https://github.com/cisagov/cert-read-role-tf-module/pull/18/commits/f13d1a4ac20e1d269620a0794928bccd9de0b79f.

Comment (mcdonnnj) [reply README.md]: Ah, ok. I found it the easiest way to update since I was touching a lot of older repos.

**Review by jsf9k — APPROVED** [review]
:zap:

**Pull request merged**